docs/source/guide/secrets-manager.rst

```diff
@@ -0,0 +1,85 @@
+.. Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.

```

Comment:
We need to add a reference to this file in the index file [here](https://github.com/boto/boto3/blob/develop/docs/source/guide/index.rst) by adding `secrets-manager` after `s3`.

Reply:
added in latest commit.